tests/integration/src/test/resources/org/mule/test/integration/transaction/transaction-expression-filter-test-config.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:vm="http://www.mulesoft.org/schema/mule/vm"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="
+       http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+       http://www.mulesoft.org/schema/mule/vm http://www.mulesoft.org/schema/mule/vm/current/mule-vm.xsd">
+
+     <vm:connector name="default"/>
+
+     <flow name="expression-in-transaction-flow">
+          <vm:inbound-endpoint path="transaction-filter-all" connector-ref="default"/>
+          <transactional action="ALWAYS_BEGIN">
```

Comment:
Does this fail only in the case of a transactional scope?

Reply:
yes